c++/src/capnp/compat/byte-stream-test.c++

```diff
@@ -58,9 +58,13 @@ KJ_TEST("KJ -> ByteStream -> KJ without shortening") {
   auto wrapped = factory1.capnpToKj(factory2.kjToCapnp(kj::mv(pipe.out)));
 
   {
-    auto promise = wrapped->write("foo", 3);
```

Comment:
Is a non-split write tested anywhere else? If not I'd leave this old case in too.

Reply:
Well, the final step of a split write is a non-split write.

Reply:
I think it would be better to add a new test case rather than edit an existing one.